src/domain/models/Pet.ts

```diff
@@ -0,0 +1,9 @@
+export enum PetGender {
+  MALE = 'M',
+  FEMALE = 'F'
+}
+
+export type Pet = {
```

Comment:
this model pet, doesn't make sense add specie too? because specie has a model too, but it is in AddPet when we put a specie inside the Pet.

Reply:
Do you think I should keep the file as `pet.ts` in the templates folder just with the enum or move it to a new file in another folder?